coretex/cli/modules/node.py

```diff
@@ -236,6 +236,32 @@ def selectNodeMode(storagePath: str) -> Tuple[int, Optional[int]]:
     return availableNodeModes[selectedMode], None
 
 
+def promptCpu(config: Dict[str, Any], cpuLimit: int) -> int:
+    cpuCount: int = clickPrompt("Enter the number of CPUs the container will use (press enter to use default)", config_defaults.DEFAULT_CPU_COUNT, type = int)
```

Comment:
Default value should be minimum of `config_defaults.DEFAULT_CPU_COUNT` and `cpuLimit`.